index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+ <head>
+  <meta charset="utf-8">
+  <title>My Portfolio</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+ </head>
+ <body>
+  <header>
+   <nav>
+    <div class="icon"><a href="">My Logo</a></div>
+    <img src="images/Union.png" alt="">
+   </nav>
```

Comment:
- [ ] As pointed out by the previous reviewer, the hamburger menu is way too big as per the design. You can adjust this by reducing the size of the `img` tag **but I strongly advise you use an icon instead of an image for this purpose. An icon will make the size blend the environment easily.
You may want to [read this for more information](https://www.w3schools.com/howto/howto_js_mobile_navbar.asp) on how to add a hamburger menu to an HTML document.
## Your Design:point_down:
![image](https://user-images.githubusercontent.com/45668340/148490678-758c8131-90fb-47b2-9528-78869328eb21.png)
## Figma Design
![image](https://user-images.githubusercontent.com/45668340/148490772-e25ffb58-979e-4940-82e6-a2ce4315e624.png)

Reply:
This well addressed  👍🏼